packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -26,6 +26,11 @@ export const AlertConsumers = {
 export type AlertConsumers = typeof AlertConsumers[keyof typeof AlertConsumers];
 export type STATUS_VALUES = 'open' | 'acknowledged' | 'closed' | 'in-progress'; // TODO: remove 'in-progress' after migration to 'acknowledged'
 
+export const ALERT_STATUS_ACTIVE = 'active';
```

Comment:
I noticed a new `alerts_as_data_severity` file was created a few days ago https://github.com/elastic/kibana/blob/master/packages/kbn-rule-data-utils/src/alerts_as_data_severity.ts

Shall we consider moving these constants to a `new alerts_as_data_status` file? It's fine if we keep things here. I was just wondering what would be the benefit of having the constants of each field in a separate file.

Reply:
Good catch, will do :+1: